keras_trainer/data_generators.py

```diff
@@ -431,6 +457,7 @@ class EnhancedDataFrameIterator(EnhancedBatchFromFilesMixin, EnhancedIterator):
         x_col: string, column in `dataframe` that contains the filenames (or
             absolute paths if `directory` is `None`).
         y_col: string or list, column/s in `dataframe` that has the target data.
+        crop_column: list, (optional) column in `dataframe` that contains the custom crop coordinates.
```

Comment:
Following the `x_col` and `y_col` terminology, should we do 
```suggestion
        crop_col: list, (optional) column in `dataframe` that contains the custom crop coordinates.
```
?

Reply:
I'm fine either way but if this change has to be made it has to be done in at least 7 or 8 places unless you think it really matters.